### PR TITLE
Feat/add comments and replies

### PR DESCRIPTION
**Pull request by Maianki**
### Added comments/reply and delete functionality
This PR includes

- [ ] Files for comments page
- [ ] Functionality and API for comments section

## Thanks for reviewing my PR

### CODE REVIEW

File: src/features/Post/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          postId,
+          commentData: { text: commentData },
+        })
+      );
+    }
+    setCommentData("");
+  };
+
+  const deleteCommentHandler = (postId, commentId) => {
+    console.log(userToken, " ", postId, " ", commentId);
```
Comment (sainath7878) [inline src/features/Post/Comment.jsx]: remove console

Comment (BalajiHarinath) [inline src/features/Post/Comment.jsx]: Can remove the log if not required

File: src/features/Post/postSlice.js [hunk 2]
```diff
@@ -101,6 +101,38 @@ export const deleteUserPost = createAsyncThunk(
...
+  "posts/getAllComments",
+  async ({ token, postId, commentData }, { rejectWithValue }) => {
+    try {
+      const response = await addCommentsService(token, postId, commentData);
+      if (response.status === 201) {
+        toast.success("comment added successfully");
+        return { comments: response.data.comments, postId };
+      }
+    } catch (error) {
+      return rejectWithValue(error.response.data);
```
Comment (sainath7878) [inline src/features/Post/postSlice.js:114]: Appreciation: Good that you're handling the errors.

File: src/features/Post/postSlice.js [hunk 3]
```diff
@@ -101,6 +101,38 @@ export const deleteUserPost = createAsyncThunk(
...
+  async ({ token, postId, commentId }, { rejectWithValue }) => {
+    try {
+      const response = await deleteCommentService(token, postId, commentId);
+
+      if (response.status === 201) {
+        toast.success("comment deleted successfully");
+        return { comments: response.data.comments, postId };
+      }
+    } catch (error) {
+      console.log(error);
```
Comment (sainath7878) [inline src/features/Post/postSlice.js]: remove console

**Review by sainath7878 — APPROVED** [review]
Functionality is working as expected. Clean and neat code. Appreciate that you've done error handling. Awesome UI.

**Review by BalajiHarinath — APPROVED** [review]
Everything looks good and nice that you are using chakra UI

File: src/features/Post/Comment.jsx (hunk 1 above)
Comment (Maianki) [reply src/features/Post/Comment.jsx]: sure will remove it.

Comment (Maianki) [reply src/features/Post/Comment.jsx]: sure will remove it.

File: src/features/Post/postSlice.js (hunk 3 above)
Comment (Maianki) [reply src/features/Post/postSlice.js]: sure will remove it.

**Pull request merged**